src/Microsoft.Health.Fhir.SqlServer.Api/Features/Filters/HttpExceptionFilterAttribute.cs

```diff
@@ -39,6 +40,11 @@ public override void OnActionExecuted(ActionExecutedContext context)
                         context.Result = new JsonResult(resultJson) { StatusCode = (int)HttpStatusCode.NotFound };
                         context.ExceptionHandled = true;
                         break;
+
+                    case RecordNotFoundException _:
```

Comment:
nit: The cases for `RecordNotFoundException` and `FileNotFoundException` are doing the same actions on the result. We can simplify the code

Reply:
Good point! Thanks